code/items/weapons/TTTWeapon.cs

```diff
@@ -104,6 +104,10 @@ public override void Spawn()
             base.Spawn();
 
             AmmoClip = ClipSize;
+            if (Owner is TTTPlayer owner)
+            {
+                (owner.Inventory as Inventory).Ammo.RegisterType(AmmoType);
```

Comment:
Do we need to run a RPC to register stuff? Can't this be done in the SetAmmo RPC (and if no ammo was registered, returning 0 ammo)?

Reply:
You are correct. RPC removed, logic already exists within SetAmmo.